chapters/en/slides/module1/module1_01.Rmd

```diff
@@ -49,268 +51,162 @@ type: slides
+70s.](https://en.wikipedia.org/wiki/Anscombe%27s_quartet)
+Can you see how the differences in the general trends between these four sets of numbers?
 
 ---
 
-## Summary statistics don't necessarily help
-
-### All four sets of numbers are nearly identical in the following statistical properties
-
-<table style="width:80%;">
-<tbody><tr>
-<th>Property
-</th>
-<th>Value
-</th>
-<th>Accuracy
-</th></tr>
-<tr>
-<td><a href="/wiki/Mean" title="Mean">Mean</a> of <i>x</i>
-</td>
-<td>9
-</td>
-<td>exact
-</td></tr>
-<tr>
-<td>Sample <a href="/wiki/Variance" title="Variance">variance</a> of <i>x</i></td>
-<td>11
-</td>
-<td>exact
-</td></tr>
-<tr>
-<td>Mean of <i>y</i>
-</td>
-<td>7.50
-</td>
-<td>to 2 decimal places
-</td></tr>
-<tr>
-<td>Sample variance of <i>y</i></td>
-<td>4.125
-</td>
-<td>±0.003
-</td></tr>
-<tr>
-<td><a href="/wiki/Correlation" class="mw-redirect" title="Correlation">Correlation</a> between <i>x</i> and <i>y</i>
-</td>
-<td>0.816
-</td>
-<td>to 3 decimal places
-</td></tr>
-<tr>
-<td><a href="/wiki/Linear_regression" title="Linear regression">Linear regression</a> line
-</td>
-<td><i>y</i>&#160;=&#160;3.00&#160;+&#160;0.500<i>x</i>
-</td>
-<td>to 2 and 3 decimal places, respectively
-</td></tr>
-<tr>
-<td><a href="/wiki/Coefficient_of_determination" title="Coefficient of determination">Coefficient of determination</a> of the linear regression</td>
-<td>0.67
-</td>
-<td>to 2 decimal places
-</td></tr></tbody></table>
+## Although summary statistics are often useful, they don't tell the whole story
+
+### C is the only set with a different mean and standard deviation
+
+```{python echo=FALSE, results='asis'}
+for set_name, df in anscombe.groupby('Set'):
+    df = df.drop(columns='Set')
+    df.columns = pd.MultiIndex.from_tuples([(set_name, 'X'), (set_name, 'Y')])
+    print(
+        df
+        .agg(['mean', 'std']).round(2).reset_index().rename(columns={'index': ''})
+        .to_html(index=False)
+        .replace('class="dataframe"', 'style="width:70px; float:left; margin-right:40px"')
+        .replace('<thead>',  '<thead style="text-align: center">'))
+```
 
 Notes:
-What about if I showed you the commonly used numerical summaries of the data?
-Tough luck,
-all the summary statistics in the table below are **the same** for each of these four set.
+Summaries,
+such as the mean and standard deviation,
+are helpful statistical tools
+that are often useful for detecting difference between datasets.
+However,
+since they collapse the data into just a few numbers,
+statistical summaries can't tell the whole story about the data
+and there can be important differences between datasets
+that summaries fail to reveal.
+Here,
+the mean and standard deviation indicate that set C
+is slightly different from the other sets of data
+in terms of the centre of the sample distribution and the spread of that distribution,
+while the remaining three sets of data have a similar centre and spread.
```

Comment:
Can you highlight the differences in C? With a red circle around the number they should first pay attention to? Or shade the background of that cell? Anything that is not too difficult to do that draws attention quickly to it would help.

Reply:
Is bolding too little? It is the easiest, but I might be able to add a color ful span, let's see 
![image](https://user-images.githubusercontent.com/4560057/101963076-aaca2200-3bc2-11eb-9160-ee8a5d015bda.png)

Reply:
I think I should also be able to get this working if you think it is better 
![image](https://user-images.githubusercontent.com/4560057/101963417-81f65c80-3bc3-11eb-89c7-07dd241a1449.png)

Reply:
Ah this is my favorite, mathcing the color to the color theme 
![image](https://user-images.githubusercontent.com/4560057/101963558-dac5f500-3bc3-11eb-85f7-ee6ce4b98b7f.png)